package.json

```diff
@@ -9,25 +9,50 @@
     "open data",
     "open knowledge"
   ],
-  "main": "index.js",
+  "main": "lib/index.js",
   "dependencies": {
-    "bluebird": "^2.9.30",
     "csv": "^0.4.2",
     "d3-time-format": "^0.3.2",
     "moment": "^2.13.0",
     "superagent": "^1.2.0",
```

Comment:
Probably same story as with native Promise? Need to check current browsers support for that technology.